Formula/deployer.rb

```diff
@@ -0,0 +1,16 @@
+class Deployer < Formula
+  desc "Deployment tool written in PHP with support for popular frameworks"
+  homepage "https://deployer.org"
+  url "https://deployer.org/releases/v6.1.0/deployer.phar"
+  sha256 "260a88bb89711cb2251496e8c353ee3be0e86cf8985117a8a714369fb35699f8"
+
+  bottle :unneeded
+
+  def install
+    bin.install "deployer.phar" => "deployer"
```

Comment:
The official website recommends it be installed under the name `dep`: https://deployer.org/docs

Reply:
`dep` collides with an existing package ( https://github.com/Homebrew/homebrew-core/blob/master/Formula/godep.rb ) and installation would fail.

Reply:
Add `conflicts_with` to both `godep` and `deployer`

Reply:
Would also conflict with [`dep`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/dep.rb)

Reply:
Done. Just wondering: why are there two packages for Go dep (`dep`/`godep`)?

Reply:
`dep` and `godep` are two separate projects.
https://github.com/golang/dep
https://github.com/tools/godep